components/menu-dropdown/menu-dropdown.jsx

```diff
@@ -941,12 +1033,15 @@ const MenuDropdown = createReactClass({
 				{...CustomTriggerChildProps}
 			/>
 		);
-	},
-});
+	}
+}
 
 MenuDropdown.contextTypes = {
 	iconPath: PropTypes.string,
 };
 
+MenuDropdown.propTypes = propTypes;
+MenuDropdown.defaultProps = defaultProps;
```

Comment:
Why do we define these above the class then reference those variables here, rather than define them inline here only once? Is that just a DSR standard?

Reply:
Yeah, artifact of history. From my point of view, even though we have a website, there are still engineers who don't trust docs and look at the source code for prop types, so putting them first helps their dev experience.